ecommerce/serializers/subcategories.py

```diff
@@ -0,0 +1,112 @@
+from rest_framework import serializers
+
+from ecommerce.models import ProductSubCategory
+
+
+class SubCategoryListSerializer(serializers.ModelSerializer):
+    """
+    Serializer for listing subcategories.
+
+    This serializer provides essential information for subcategory listings,
+    optimized for performance and minimal data transfer.
+    """
+
+    category = serializers.SerializerMethodField()
+
+    class Meta:
+        model = ProductSubCategory
+        fields = [
+            "id",
+            "name",
+            "slug",
+            "description",
+            "category",
+            "is_active",
+            "created_at",
+            "updated_at",
+        ]
```

Comment:
No need to expose the meta field in the response.
- `created_at`
- `updated_at`
- `description`